tests/functional/modules/test_zos_job_output_func.py

```diff
@@ -100,10 +97,11 @@ def test_zos_job_output_job_exists(ansible_zos_module):
         )
 
         jobs = hosts.all.zos_job_submit(
-            src="{0}/SAMPLE".format(TEMP_PATH), location="USS", wait=True, volume=None
+            src="{0}/SAMPLE".format(TEMP_PATH), location="USS", volume=None
         )
-
+        sleep(30)
```

Comment:
I do recall that this was needed because we thought we needed to wait to get the dd_name properly, but after we knew it was because the schema changed, should we keep this one here? If so, please add a comment stating the reason behind this